myapp/app/views/tasks/index.html.erb

```diff
@@ -5,11 +5,13 @@
     </h1>
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
       <ul class="navbar-nav me-auto mb-2 mb-lg-0"></ul>
-      <div class="task_navbar__log_out_btn">
-        <%= link_to t('pages.navbar.admin'),
-                    users_path,
-                    class: 'nav-link active' %>
-      </div>
+      <% if logged_in? && admin? %>
+        <div class="task_navbar__log_out_btn">
+          <%= link_to t('pages.navbar.admin'),
+                      users_path,
+                      class: 'nav-link active' %>
```

Comment:
[FYI]

In Rakuma, We do not use `i18n.t` as an abbreviation for `t` because we want to be explicit about which method we are using! However, this is not a problem during training.